packages/editor/src/bindings/post-meta.js

```diff
@@ -82,19 +131,14 @@ export default {
 		return true;
 	},
 	getFieldsList( { registry, context } ) {
-		const metaFields = registry
-			.select( coreDataStore )
-			.getEditedEntityRecord(
-				'postType',
-				context?.postType,
-				context?.postId
-			).meta;
+		const metaFields = getMetadata( registry, context );
 
 		if ( ! metaFields || ! Object.keys( metaFields ).length ) {
 			return null;
 		}
 
 		// Remove footnotes or private keys from the list of fields.
+		// TODO: Remove this once we retrieve the fields from 'types' endpoint in post or page editor.
```

Comment:
Will this be handled separately, and is there an open PR for it?

Reply:
There is no PR yet. We managed to not touch any REST API related code. I don't know if it will be done for 6.7.

Reply:
Would it make sense to move this filtering to `getMetadata` helper? Are the fields with `_` prefix included in the schema since they are considered private?